src/content/services/monetizationTagManager.ts

```diff
@@ -94,7 +89,7 @@ export class MonetizationTagManager extends EventEmitter {
   }
 
   private async resumeAllMonetization() {
-    const response = await isWMEnabled()
+    const response = await this.message.send('IS_WM_ENABLED', undefined)
```

Comment:
Look if we can avoid passing undefined as the second parameter (the payload).

Reply:
Done in https://github.com/interledger/web-monetization-extension/pull/534/commits/2c178a023bf5c175caf12b548b4cd7c2e839bf63.